webapp/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultCsvValidationService.java

```diff
@@ -116,9 +105,7 @@ public DefaultCsvValidationService(final SchoolRepository schoolRepository) {
             if (currentSchool == null || !currentSchool.equals(school)) {
                 if (currentSchool != null) {
                     completedSchools.add(currentSchool);
-                    if(failures.isEmpty()) {
```

Comment:
This if condition can never be false because of the above ```if (!failures.isEmpty()) continue;``` statement

Reply:
Yeah, I question `!failures.isEmpty()` as it doesn't report all failures in that case.  I thought the original intent was for it to report all failures from Will's demo.  I think removing this defensive check can cause issues later should we decide again to report all failures.

Reply:
Well, it doesn't belong in the current implementation. I can add a comment though in place of the duplicate if check